src/components/pages/Dashboard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import PageTitle from '../atoms/PageTitle';
+import Tab from '../molecules/Tab';
+import TabsContainer from '../molecules/TabsContainer';
+
+interface IProps {
+  className?: string;
+}
+
+const UnstyledDashboard: React.FC<IProps> = props => {
+  return (
+    <div className={props.className}>
+      <div className="page-title">
+        <PageTitle
+          title="Dashboard"
+          description="Showing key performance indicators for current company"
+        />
+      </div>
+      <TabsContainer className="tabs-container">
+        <Tab label="Minimum Liquidity">
+          <h3>Projected minimum liquidity</h3>
+        </Tab>
+        <Tab label="Transaction Volumes">
```

Comment:
This implementation requires that all label props are unique. This shouldn't be a problem as using two equal tablabels wouldn't make any sense in a tabbed view, but we might want to enforce this somehow. Would like some input/feedback on that.